test/extended/networking/commatrix.go

```diff
@@ -0,0 +1,237 @@
+package networking
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"slices"
+	"strconv"
+	"strings"
+
+	"os"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift-kni/commatrix/pkg/types"
```

Comment:
Commatrix vendoring, like the other pr we had to revert?

Is it possible that something goes wrong, and requires a fix in commatrix, then vendoring, to restore passing tests?